sympy/logic/boolalg.py

```diff
@@ -326,6 +326,73 @@ def _new_args_filter(cls, args):
             newargs.append(x)
         return LatticeOp._new_args_filter(newargs, And)
 
+    def as_solution_set(self, *args):
+        """
+        Rewrite the solution as logic and relationals from
+        solve_multivariate (NotImplemented) in form of real sets.
+
+        Assumptions
+        ===========
+        TODO - conclusion after discussion
+
+        Examples
+        ========
+
+        >>> from sympy import And, Symbol
+        >>> x = Symbol('x', real=True)
+        >>> And(x<2, x>-2).as_solution__set()
+        (-2, 2)
+
+        For multivariate logical expressions:
+
+        >>> a = And(x>0, y>x)
+
+        >>> a.as_solution_set(x, y)
+        (0, oo) x (x, oo)
+        >>> a.as_solution_set(y, x)
+        (-oo, oo) x ((-oo, y) ∩ (0, oo))
+        In the above example as y is passed before x, so it means
+        if required x has to be expressed in terms of y which results
```

Comment:
I don't understand what this sentence is trying to say. Why wouldn't switching the arg order just switch the order of the resulting sets?